bodhi/tests/server/base.py

```diff
@@ -40,7 +40,7 @@
 # https://wiki.jenkins-ci.org/display/JENKINS/Building+a+software+project#Buildingasoftwareproject-below
 if os.environ.get('BUILD_ID'):
     try:
-        req = requests.get('%s/new' % FAITOUT)
+        req = requests.get('%s/new' % FAITOUT, timeout=60)
```

Comment:
I presumed that the unit here is seconds, but the requests docs do not say:

http://docs.python-requests.org/en/master/api/#requests.request

Also, 60 seconds is an arbitrary value I pulled out of nowhere.